src/gdcdictionary/schemas/pathology_detail.yaml

```diff
@@ -236,6 +236,14 @@ properties:
         $ref:
           - "_terms_enum.yaml#/not_reported/common"
 
+  days_to_pathology_detail:
+    description: >-
```

Comment:
should we put the description in _terms file? ditto `micrometastasis_present` below

Reply:
Yeah we need to flesh out a process for what to do in terms definition adding when we do not have a code in a terms database (NCIt, caDSR etc) but still want to offer a description, especially for properties. I think for now we're ok but if/when the EVS team hopefully generates a code for this we can formally add to terms file with associated definition ID.